go.mod

```diff
@@ -32,19 +32,19 @@ require (
 	go.opentelemetry.io/otel/sdk v1.4.0
 	go.opentelemetry.io/otel/sdk/metric v0.27.0
 	go.opentelemetry.io/otel/trace v1.4.0
-	golang.org/x/crypto v0.0.0-20211117183948-ae814b36b871
-	golang.org/x/mod v0.5.1
+	golang.org/x/crypto v0.0.0-20220214200702-86341886e292
+	golang.org/x/mod v0.6.0-dev.0.20220106191415-9b9b3d81d5e3
 	helm.sh/helm/v3 v3.8.1
-	k8s.io/api v0.23.5
-	k8s.io/apimachinery v0.23.5
-	k8s.io/client-go v0.23.5
-	k8s.io/component-base v0.23.5
-	k8s.io/klog/v2 v2.30.0
-	k8s.io/kubectl v0.23.4
-	k8s.io/utils v0.0.0-20211116205334-6203023598ed
-	sigs.k8s.io/cluster-api v1.1.1
+	k8s.io/api v0.24.2
+	k8s.io/apimachinery v0.24.2
+	k8s.io/client-go v0.24.2
+	k8s.io/component-base v0.24.2
+	k8s.io/klog/v2 v2.60.1
+	k8s.io/kubectl v0.24.2
+	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+	sigs.k8s.io/cluster-api v1.1.4
 	sigs.k8s.io/cluster-api/test v1.1.4
-	sigs.k8s.io/controller-runtime v0.11.2
+	sigs.k8s.io/controller-runtime v0.12.1
```

Comment:
did this get bumped automatically? We should be synced with CAPI, and CAPI v1.1.4 still uses v0.11.2 https://github.com/kubernetes-sigs/cluster-api/blob/v1.1.4/go.mod#L45